tests/Filesystem/FilesystemTest.php

```diff
@@ -339,7 +339,7 @@ public function testSharedGet()
             $result *= $status;
         }
 
-        $this->assertTrue($result === 1);
+        $this->assertFalse($result === 1);
```

Comment:
file_put_contents does not support share lock and the lock parameter will not work.

Thus, it may result in some processes are reading data when other process are writing in the same time.